email/email.go

```diff
@@ -11,27 +11,32 @@ import (
 
 // Client Class containing the needed methods to send an email
 type Client struct {
-	apiKey string
+	sendgridClient *sendgrid.Client
 }
 
 // NewClient class constructor function
 func NewClient(apiKey string) (*Client, error) {
 	if len(apiKey) <= 0 {
 		return nil, errors.New("Api Key should not be empty")
 	}
-	return &Client{apiKey}, nil
+
+	client := sendgrid.NewSendClient(apiKey)
```

Comment:
Could inline sendgrid.NewSendClient(apiKey) in the the row below.

Reply:
Nice! Done